From 98d86729da341c82f4776145511d01e8bfdfed29 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 21 Jan 2022 08:49:13 +0000
Subject: [PATCH] arm: Remove spurious .fnend directive

...a.k.a. Fix annoying build error

Some toolchains rightly object to the fact that once the acros are
expanded this results in multiple ".fnend"s without ".fnstart"s.

See: https://github.com/raspberrypi/linux/issues/4836

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/lib/memcpymove.h | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm/lib/memcpymove.h
+++ b/arch/arm/lib/memcpymove.h
@@ -280,7 +280,6 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBI
 199:
         pop     {DAT3, DAT4, DAT5, DAT6, DAT7}
         pop     {D, DAT1, DAT2, pc}
-        UNWIND( .fnend )
 .endm
 
 .macro memcpy_medium_inner_loop  backwards, align
